### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ New Project_ Color Contrast: Color Contrast ratio for “Maven” menu item is less than 4.5:1.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3355]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** 1.4.3

**Repro Steps:**
1) Open IntelliJ IDEA application.
2) Press 'Ctrl+ Alt+ S' to open application settings/preferences popup.
3) Tab till 'Plugins' tab and press enter.
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter.
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in.
7) Sign in with any Azure subscription.
8) Tab till File menu and click new-->Project.
9) In the New Project dialog box, select Maven.
10)Verify Color contrast ratio for “Maven”.

**Actual Result:**
Color Contrast ratio for “Maven” menu item is less than 4.5:1.

**Expected Result:**
Color Contrast ratio for “Maven” menu item should be grater than 4.5:1.

**User Impact:**
Low vision users can not see complete information if colour contrast ratio is not proper.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![Bug_3355_Color Contrast issue](https://user-images.githubusercontent.com/38523250/62207727-847f5a00-b3b2-11e9-869b-7618e9655631.png)

**Comment by andxu** [comment]
Hi @Accessibility56 ,this bug is very similar to https://github.com/microsoft/azure-tools-for-java/issues/3356, the maven tab in this issue is provided by IntelliJ idea which is out of our control, would you please help us to create a bug here at https://youtrack.jetbrains.com/issues, thanks.

**Comment by andxu** [comment]
* Importance of experience: Little, user can use high contrast theme to solve this issue.
* Quality of experience: Little, high contrast theme is suggested by idea for users with visual impairment 
 Alternate solutions: Change theme in idea.

![image](https://user-images.githubusercontent.com/17075863/64594522-cc969100-d3e2-11e9-94a6-087e079e9acf.png)

**Comment by Accessibility56** [comment]
@andxu  Please provide the link to AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Accessibility56** [comment]
Logged an External bug here: https://youtrack.jetbrains.com/issue/IDEA-222347

**Comment by Flanker32** [comment]
@Accessibility56 Issue tracked in AAT, please help us review it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749789
